aten/src/ATen/function_wrapper.py

```diff
@@ -229,7 +229,9 @@ def TypedDict(name, attrs, total=True):  # type: ignore
 
 CALL_TEMPLATE = CodeTemplate("${cname}(${actuals})")
 
-OPERATOR_NAME = CodeTemplate("""\
+OPERATOR_NAME = CodeTemplate("aten::${operator_name}")
+
+OPERATOR_NAME_FULL = CodeTemplate("""\
     {"aten::${operator_name}", "${overload_name}"},
```

Comment:
So if I understand correctly, per-op registration doesn't give a separate file to each overload?

Reply:
Yes, the current granularity is at operator level. The code analyzer doesn't yet extract overload name from op registration/invocation. We could break it down if overload name is widely used and has meaningful impact on custom build size.

Reply:
That's fine, it just wasn't obvious to me that this is what was happening looking at the diff.